corehq/apps/case_importer/tracking/dbaccessors.py

```diff
@@ -29,7 +31,11 @@ def get_case_upload_record_count(domain, user):
 
 def get_case_ids_for_case_upload(case_upload):
     for form_record in case_upload.form_records.order_by('pk').all():
-        form = XFormInstance.objects.get_form(form_record.form_id, case_upload.domain)
+        if toggles.DOMAIN_PERMISSIONS_MIRROR.enabled(case_upload.domain) \
```

Comment:
You don't need to check the flag, that's just there to control access to the enterprise permissions config UI.

Reply:
I see, I misinterpreted this. So if the enterprise permissions are mirrored on another domain you can simply check the permissions on that domain.